File: task3.kt
fun isPalindrome(num: Int): Boolean {
    val numStr = num.toString()
    return numStr == numStr.reversed()
}

fun main() {
    print("Enter the numbers with a space: ")
    val entryNums = readLine()
    val nums = entryNums?.split(" ") ?: emptyList()

    for (num in nums) {
        if (num.length != 3) {
            println("$num - It's not a three-digit number")
            continue
        }
        val numInt = num.toInt()
        if (isPalindrome(numInt)) {
            println("$num - Is Palindrome")
        } else {
            println("$num - Is not Palindrome")
        }
    }
}

File: Task3.swift
func isPalindrome(_ num: Int) -> Bool {
    let numStr = String(num)
    return numStr == String(numStr.reversed())
}

print("Enter the numbers with a space:")
if let entryNums = readLine() {
    let nums = entryNums.split(separator: " ")
    
    for num in nums {
        if num.count != 3 {
            print("\(num) - It's not a three-digit number")
            continue
        }
        
        if let numInt = Int(num) {
            if isPalindrome(numInt) {
                print("\(numInt) - Is Palindrome")
            } else {
                print("\(numInt) - Is not Palindrome")
            }
        }
    }
}
